homeassistant/components/device_tracker/ee_brightbox.py

```diff
@@ -57,6 +112,7 @@ def check_config(self):
             _LOGGER.exception("Failed to connect to the router")
             return False
 
+    @Throttle(SCAN_INTERVAL)
     def scan_devices(self):
```

Comment:
I think you caught this already, but need to handle when new devices are found outside setup.